master/static/migrations/20240319162250_update-stream-project-change-func.tx.up.sql

```diff
@@ -0,0 +1,78 @@
+CREATE OR REPLACE FUNCTION stream_project_change() RETURNS TRIGGER AS $$
```

Comment:
If I remember correctly, Nick has moved all sql functions out of migration files. I will update this.

Reply:
can you rebase so i can get a diff instead (very excited for this change)